src/types.ts

```diff
@@ -24,11 +24,11 @@ export enum LetterCasing {
   lowercase = 'lowercase'
 }
 
-export type StyledValue = string | 0 | number;
+export type StyledValue = string | number | undefined;
 
 export type StyledObject = { [property: string]: StyledValue };
 
-export type FontSources = { [K in FontSourceFormats]?: string | string[] | NodeRequireFunction };
+export type FontSources = { [K in FontSourceFormats]?: string | string[] | NodeRequireFunction | false };
```

Comment:
I solved this upstream by using a ternary instead of `&&`, but sure, I don't mind accepting `false` here.